identity-iota/src/tangle/message_history.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::chain::DiffChain;
+use crate::chain::IntegrationChain;
+use crate::did::DocumentDiff;
+use crate::did::IotaDID;
+use crate::did::IotaDocument;
+use crate::error::Result;
+use crate::tangle::Client;
+use crate::tangle::Message;
+use crate::tangle::MessageId;
+use crate::tangle::MessageSet;
+
+/// A representation of a DID Document message history.
+#[derive(Clone, Debug, Serialize)]
+pub struct MessageHistory {
+  #[serde(rename = "intChainData")]
+  pub int_chain_data: IntegrationChain,
+  #[serde(rename = "intChainSpam")]
+  pub int_chain_spam: Option<Vec<MessageId>>,
+  #[serde(rename = "diffChainData")]
```

Comment:
Is it worth mentioning here that the it only queries the "lastestDiffChainData" here? May be implied already.